app/src/main/java/com/architectcoders/aacboard/ui/fragments/EditDashboardFragment.kt

```diff
@@ -30,7 +30,14 @@ class EditDashboardFragment : Fragment() {
     private val viewModel: EditDashBoardViewModel by viewModel {
         parametersOf(args.dashBoardId)
     }
-    private val dashboardCellsAdapter = DashboardCellsAdapter { }
+    private val dashboardCellsAdapter = DashboardCellsAdapter { cell ->
```

Comment:
for consistency i would move this to an state holder as the in the other fragments